deps/lua/src/loslib.c

```diff
@@ -234,10 +234,17 @@ static const luaL_Reg syslib[] = {
 
 /* }====================================================== */
 
+#define UNUSED(V) ((void) V)
 
+/* Only a subset is loaded currently, for sandboxing concerns. */
+static const luaL_Reg sandbox_syslib[] = {
+  {"clock",     os_clock},
+  {NULL, NULL}
+};
 
 LUALIB_API int luaopen_os (lua_State *L) {
-  luaL_register(L, LUA_OSLIBNAME, syslib);
+  UNUSED(syslib);
```

Comment:
Do we need this? Compiler will raise a warning on an unused static const var?

Reply:
yes, and I can't delete or comment `syslib` out,  otherwise, will raise a lot of unused method warning.